fleet_vehicle_service_activity/models/fleet_vehicle_log_services.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class FleetVehicleLogServices(models.Model):
+    _inherit = "fleet.vehicle.log.services"
+
+    @api.model
+    def scheduler_manage_service_date(self):
```

Comment:
Use method name convention:

```suggestion
    def _cron_manage_service_date(self):
```